Makefile

```diff
@@ -155,6 +157,7 @@ QEMU_SYSTEM_arm64=qemu-system-aarch64
 QEMU_SYSTEM_amd64=qemu-system-x86_64
 QEMU_SYSTEM=$(QEMU_SYSTEM_$(ZARCH))
 
+#QEMU_ACCEL_Y_Darwin_arm64=-machine virt,accel=hvf,usb=off -cpu host,kvmclock=off
```

Comment:
I don't think you should be commenting this out @vmlemon

Reply:
There's no support for Hypervisor.framework acceleration, on Darwin/ARM64, in QEMU, right now - so, it will bail out, and refuse to launch, claiming that it's an unsupported acceleration technology, until someone fixes upstream QEMU.

Reply:
(On my machine, it says "```tyson@Tysons-Air ~ % ./qemu-6.0.0/build/qemu-system-aarch64 -machine virt,accel=hvf,usb=off -cpu host,kvmclock=off
qemu-system-aarch64: invalid accelerator hvf```")

Reply:
Ah! Ok.